cms/static/sass/views/_group-configuration.scss

```diff
@@ -446,6 +446,45 @@
           margin-bottom: 0;
         }
       }
+
```

Comment:
The majority of this styling code is being duplicated from what it necessary to style experiment groups. It would be cleaner for this to be generic styling defined once in this file, with any potential overrides of the css for the specific cases added to the "// specific group-type styles" areas inside of either .content-groups or .experiment-groups. If the styles are 100% identical then there won't even need to be any overrides defined at all.